src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po

```diff
@@ -0,0 +1,273 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"POT-Creation-Date: 2022-12-16 02:02+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI +ZONE\n"
+"Last-Translator: Julien Chandelle <jchandelle@affinitic.be>\n"
+"Language-Team: French <FR@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0\n"
+"Language-Code: fr\n"
+"Language-Name: French\n"
+"Preferred-Encodings: utf-8 latin1\n"
+"Domain: plone.restapi\n"
+
+#: plone/restapi/services/email_send/post.py:81
+msgid "${sender_fullname} via ${portal_title}"
+msgstr "${sender_fullname} via ${portal_title}"
+
+#: plone/restapi/services/email_send/post.py:74
+msgid "A portal user via ${portal_title}"
+msgstr "Un portail utilsateur via ${portal_title}"
+
+#: plone/restapi/configure.zcml:73
+msgid "Adds sample content for performance testing"
+msgstr "Ajoute un exemple de contenu pour les tests de performances"
+
+#: plone/restapi/configure.zcml:56
+msgid "Adds sample content types for testing"
+msgstr "Ajoute des exemples de types de contenu pour les tests"
+
+#: plone/restapi/configure.zcml:65
+msgid "Adds sample workflows for testing"
+msgstr "Ajoute des exemples processus pour les tests"
+
+#: plone/restapi/services/aliases/add.py:106
+msgid "Alternative urls that point to themselves will cause an endless cycle of redirects."
+msgstr "Les URL alternatives qui pointent vers elles-mêmes provoqueront un cycle sans fin de redirections."
+
+#: plone/restapi/configure.zcml:115
+msgid "Blocks"
+msgstr "Blocs"
+
+#: plone/restapi/configure.zcml:122
+msgid "Blocks (Editable Layout)"
+msgstr "Blocs (mise en page modifiable)"
+
+#: plone/restapi/configure.zcml:122
+msgid "Enables Volto Blocks (editable layout) support"
+msgstr "Active la prise en charge des blocs Volto (mise en page modifiable)"
+
+#: plone/restapi/configure.zcml:115
+msgid "Enables Volto Blocks support"
+msgstr "Active la prise en charge des blocs Volto"
+
+#: plone/restapi/configure.zcml:81
+msgid "Enables blocks on the Document content type"
+msgstr "Active les blocs sur le type de contenu Document"
+
+#: plone/restapi/services/contextnavigation/get.py:133
+msgid "Enter a valid scale name (see 'Image Handling' control panel) to override (e.g. icon, tile, thumb, mini, preview, ... ). Leave empty to use default (see 'Site' control panel)."
+msgstr "Saisissez un nom valide d'échelle (cf. « Gestion des images » dans le panneau de configuration) pour outrepasser (p. ex. icône, tuile, miniature, mini, prévisualisation, etc.). Laisser vide pour utiliser les échelles par défaut (cf. « Site » dans le panneau de configuration)."
```

Comment:
scale names should remain in english (icon, tile, thumb, ...)

Reply:
Fix